sys/shell/commands/sc_gnrc_netif.c

```diff
@@ -1021,9 +1106,24 @@ static int _netif_set(char *cmd_name, kernel_pid_t iface, char *key, char *value
     if ((strcmp("addr", key) == 0) || (strcmp("addr_short", key) == 0)) {
         return _netif_set_addr(iface, NETOPT_ADDRESS, value);
     }
+    else if (strcmp("appeui", key) == 0) {
+        return _netif_set_lw_key(iface, NETOPT_LORAWAN_APPEUI, value);
+    }
+    else if (strcmp("appkey", key) == 0) {
+        return _netif_set_lw_key(iface, NETOPT_LORAWAN_APPKEY, value);
+    }
     else if (strcmp("addr_long", key) == 0) {
         return _netif_set_addr(iface, NETOPT_ADDRESS_LONG, value);
     }
+    else if (strcmp("deveui", key) == 0) {
+        return _netif_set_addr(iface, NETOPT_ADDRESS_LONG, value);
+    }
```

Comment:
What's wrong with using `addr_long` for that?

Reply:
technically the DEVEUI is not an address, although in RIOT is modeled using NETOPT_ADDR_LONG.

Reply:
how to proceed? Should I add a NETOPT_DEVEUI option, use `addr_long` or leave it as it is?

Reply:
Well if you feel better with that alias I won't object - but only make it available for `MODULE_GNRC_NETIF_CMD_LORA` to avoid confusion.

Reply:
I think it's better to leave it that way.

At least for the LoRaWAN user, it's usually about "deveui", "appkey" and "appeui". For ABP, "nwskey", "appskey" and the only thing I'm mapping is "addr" to "devaddr" (since I don't think someone would be confused, since it's strictly an address).

I already addressed the `#ifdef` wrap :)

Reply:
The thing is in the review this options were introduced, we came to the conclusion, that `NETOPT_DEVEUI` is the same as `NETOPT_ADDRESS_LONG`, so I would be against introducing an alias. I prefer the duplicate functionality here, as it is what a typical LoRA user would expect in the frontend.